bttc_rr_registration.html

```diff
@@ -0,0 +1,323 @@
+
+  <style>
+    body {
+      background-color: #EBE9E4;
+    }
+
+    .container {
+      background: #fff;
+      margin: 4rem auto;
+      padding: 2rem;
+      border-radius: 12px;
+      box-shadow: 0 8px 16px rgba(0, 0, 0, 0.1);
+      max-width: 500px;
+      width: 90%;
+      text-align: center;
+    }
+
+    form {
+      display: flex;
+      flex-direction: column;
+      gap: 1rem;
+      margin-top: 1rem;
+    }
+
+    input[type="text"] {
+      padding: 0.75rem;
+      font-size: 1rem;
+      border: 1px solid #ccc;
+      border-radius: 8px;
+      width: 100%;
+      box-sizing: border-box;
+    }
+
+    button {
+      padding: 0.75rem;
+      background-color: #0d6efd;
+      color: white;
+      border: none;
+      border-radius: 8px;
+      font-size: 1rem;
+      cursor: pointer;
+    }
+
+    button:hover {
+      background-color: #0b5ed7;
+    }
+
+    .entry {
+      border: 1px solid #ddd;
+      background: #fafafa;
+      padding: 1rem;
+      border-radius: 8px;
+      margin-top: 1rem;
+      text-align: left;
+    }
+
+    .entry p {
+      margin: 0.5rem 0;
+    }
+
+    .result {
+      margin-top: 1.5rem;
+    }
+
+    .error {
+      color: #dc3545;
+      font-weight: bold;
+    }
+
+    .success {
+      color: #198754;
+      font-weight: bold;
+    }
+
+    .roster-link {
+      font-weight: bold;
+      color: #0d6efd;
+      text-decoration: underline;
+      margin-top: 0.5rem;
+      display: inline-block;
+    }
+
+    .token-field {
+      width: 70%;
+      margin-top: 0.5rem;
+    }
+
+    .confirm-btn {
+      margin-top: 0.5rem;
+    }
+
+    .token-error {
+      display: block;
+      margin-top: 0.5rem;
+      color: red;
+      font-size: 0.9em;
+    }
+  </style>
+</head>
+
+<body>
+
+  <div id="header">
+    <div class="container_12 clearfix">
+      <div id="logo" class="grid_6">
+        <a href="index.html" title="Go Home">
+          <img src="images/logo.gif" height="60" alt="BTTC Logo" />
+        </a>
+      </div>
+      <div id="search" class="grid_6"></div>
+    </div>
+    <div class="border">
+      <ul id="nav" class="container_12 clearfix">
+        <li class="grid_2"><a href="index.html" title="Home">Home</a></li>
+        <li class="grid_2"><a href="results.html" title="Weekly Results">Results</a></li>
+        <li class="grid_2"><a href="coaching.html" title="Coaching">Coaching</a></li>
+        <li class="grid_2"><a href="about.html" title="About Us">About Us</a></li>
+        <li class="grid_2"><a href="club_policy.html" title="Club Policy">Club Policy</a></li>
+        <li class="grid_2 last"><a href="more.html" title="More...">More...</a></li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="container" id="lookupContent">
+    <h3>Round Robin Sign-Up</h3>
+    <p>Enter your phone number (e.g., 5101234567):</p>
+    <form id="lookupForm">
+      <input type="text" id="phoneInput" name="phone" placeholder="Enter phone number" required />
```

Comment:
```suggestion
      <input type="tel" id="phoneInput" name="phone" placeholder="Enter phone number" maxlength="16" required />
```

`type="tel"` will give the user a nicer keyboard input

![IMG_9325](https://github.com/user-attachments/assets/19b5abb4-2599-4036-9dbc-f36d66a48622)


`maxlength` should help ensure a bad actor couldn't send long character strings to try and hack the API